jest.config.js

```diff
@@ -16,5 +16,8 @@ module.exports = {
       "stringifyContentPathRegex": "\\.html$"
     }
   },
-  moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, { prefix: '<rootDir>' })
+  moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, { prefix: '<rootDir>' }),
+  transformIgnorePatterns: [
+      "node_modules/(?!@ngrx|(?!deck.gl)|ng-dynamic)"
```

Comment:
This was something I realized, but I initially was dealing with a rather absurd problem: somehow adding that string caused some Jest errors to disappear even though we didn't use any of these packages. I tried modifying the line and it would cause the errors to appear again so I kept the line like that and continued working on it.

Upon removing that line now it still works, like one would expect it to in the first place. Which is good but I'm not entirely sure why that line ever fixed anything earlier in the process. Anyway, line is removed now!